### ISSUE TITLE
AntdDatePicker shows "invalid date"

### ISSUE THREAD

**Issue by OGordon100** [CNFeffery/feffery-antd-components#51]
Using the month picker, en locale., and display format YYYY-MM. when the picker is created with a preselected value (e.g "2022-03", the picker will display "invalid date" and nothing is selectable

**Comment by CNFeffery** [comment]
@OGordon100 which version are you using.

**Comment by CNFeffery** [comment]
Fixed in [0.2.0rc22](https://pypi.org/project/feffery-antd-components/0.2.0rc22/)

```Python
fac.AntdDatePicker(
    format='YYYY-MM',
    value='2023-03',
    picker='month',
    locale='en-us'
)
```

![image](https://user-images.githubusercontent.com/49147660/215373216-4e2e6612-a6fe-4635-beee-5584b09ff7b1.png)
